MapboxStaticTests/MapboxStaticTests.swift

```diff
@@ -370,7 +370,7 @@ class MapboxStaticTests: XCTestCase {
         let fillColor = Color.red
         let fillColorRaw = "ff0000"
         let fillOpacity = 0.25
-        let encodedPolyline = "(upztG%60jxkVn@al@bo@nFWzuAaTcAyZen@)"
+        let encodedPolyline = "(upztG%60jxkVn@al@bo@pFWzuAaTcAyZgn@)"
```

Comment:
@tmcw, should we be concerned that there were differences in how Polyline encoded the polyline versus our homegrown encoder’s version? In this case at least, there didn’t appear to be any visual difference.